conscious/chain.json

```diff
@@ -0,0 +1,106 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "cvn",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://cvn.io/",
```

Comment:
should this be https://www.consciousdao.com/? And then cvn.io moved down to explorers?